<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectiveColorAdjustForm</class>
 <widget class="QWidget" name="SelectiveColorAdjustForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>252</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>青色：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="cyanEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="cyanSlider">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid;
border-top-color:#767676;
border-bottom-color:#EFEFEF;
border-left-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
border-right-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>131</y>
     <width>252</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>洋红：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="magentaEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="magentaSlider">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid;
border-top-color:#767676;
border-bottom-color:#EFEFEF;
border-left-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
border-right-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>252</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>黄色：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="yellowEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="yellowSlider">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid;
border-top-color:#767676;
border-bottom-color:#EFEFEF;
border-left-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
border-right-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>252</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>黑色：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="blackEdit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="blackSlider">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid;
border-top-color:#767676;
border-bottom-color:#EFEFEF;
border-left-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
border-right-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A6A6A6, stop:0.75 #C2C2C2, stop:1 #A6A6A6);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>21</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>红色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绿色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>蓝色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>洋红</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>白色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中性色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑色</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>24</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>颜色(O)：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>253</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>方法：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="relBtn">
      <property name="minimumSize">
       <size>
        <width>89</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>相对(R)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="absBtn">
      <property name="minimumSize">
       <size>
        <width>89</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>绝对(A)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
